terraform/aks.tf

```diff
@@ -21,14 +22,14 @@ resource "azurerm_kubernetes_cluster" "example" {
 }
 
 output "client_certificate" {
-  value = azurerm_kubernetes_cluster.example.kube_config.0.client_certificate
+  value = var.skip_pachyderm_deploy == "" ? azurerm_kubernetes_cluster.example[0].kube_config.0.client_certificate : ""
```

Comment:
How does this work if the user's cluster already exists? Is this the client certificate that authenticates the syncer with k8s? Basically the user passing their k8s client credentials to the sync?

Reply:
*Update* per slack, the user passes their client credentials to the syncer with in syncer.sh: https://github.com/pachyderm/aml/pull/62/files#diff-ce435f38623d032d25670f2c63a430dfd1a02db661d10c1ef031cbef2f0f0ef0R18

I'm still not sure how `client_certificate` is used, but it seems like `kube_config` is the TF output that writes to `terraform/kubeconfig` in `setup.sh` (which `setup.sh subsequently `scp`s to the syncer)